### ISSUE TITLE
Dependency issue for appboy

### ISSUE THREAD

**Issue by mmdock** [Appboy/appboy-ios-sdk#190]
* [X] I have checked [Braze's documentation](https://www.braze.com/documentation/iOS/) and am still experiencing the issue.

# Report
Trying to update my SDWebImage version to the 5.0.2 release, however cocoapods is having a conflict with resolving AppBoy dependencies

## Describe your environment.

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name               | iOS
 Platform Version            | N/A
 SDK Version                   | 3.14.0
 Integration Method       | cocoapods
 Xcode Version               | Version 10.2.1 (10E1001)
 Repro rate                      | 100% 
 cocoapods version        | 1.5.3 
 
## What did you do?
Attempted to update from

`   pod 'SDWebImage/WebP', '~> 4.4.2'`

to 

```
    pod 'SDWebImage', '~> 5.0.2'
    pod 'SDWebImageWebPCoder', '~> 0.2.0'
```

## What did you expect to happen?

Braze SDK is integrated properly

## What happened instead?

Getting AppBoy/Braze dependency error:
```
[!] CocoaPods could not find compatible versions for pod "SDWebImage/GIF":
  In Podfile:
    Appboy-iOS-SDK (~> 3.14.0) was resolved to 3.14.0, which depends on
      Appboy-iOS-SDK/UI (= 3.14.0) was resolved to 3.14.0, which depends on
        Appboy-iOS-SDK/NewsFeed (= 3.14.0) was resolved to 3.14.0, which depends on
          SDWebImage/GIF (~> 4.0)
Specs satisfying the `SDWebImage/GIF (~> 4.0)` dependency were found, but they required a higher minimum deployment target.
```

**Comment by Bucimis** [comment]
Hi @mmdock,

The currently released version of the SDK depends on SDWebImage 4.x. We'll be adding 5.x support in the near future, but in the meantime you'll need to continue to use 4.x.

**Comment by nebillo** [comment]
We are having the same problem in our project too, and it's quite frustrating. Can you provide a better ETA on when support for SDWebImage 5.x is expected to happen? thank you.

**Comment by flikQ** [comment]
I would also love some information on when 5.0 will be added. While paying thousands per month for your service I would greatly not like it to continue to block our development (a lot of our app is image heavy so keeping up with SDWebImage improvements is crucial.

**Comment by Bucimis** [comment]
@flikQ thanks for the feedback and our apologies for your frustration. Our 5.0 release candidate is in QA and we expect a release at the latest by end of week next week. Let us know if that aligns with your timelines.

**Comment by Bucimis** [comment]
@flikQ @mmdock @nebillo thank you for your input and patience - we've just released 3.15.0 with SDWebImage 5 support.

**Comment by flikQ** [comment]
I don't believe this is fully solved.

I have updated the SDK to this new version, however when running pods it fails as react-native-appboy-sdk is version 1.12.0 which relies on 3.14.0

> In Podfile:
    react-native-appboy-sdk (from `../node_modules/react-native-appboy-sdk`) was resolved to 1.12.0, which depends on
      Appboy-iOS-SDK (~> 3.14.0)

**Comment by Bucimis** [comment]
Hi @flikQ, that's correct, to use 3.15.0 through React Native we'll need to update our React Native wrapper to use the new release as well (it currently pulls in 3.14.0). We'll have that out shortly.

**Comment by Bucimis** [comment]
@flikQ we just released react native sdk 1.13.0, which uses ios sdk 3.15.0. Let us know how it goes!

**Comment by flikQ** [comment]
@Bucimis 
Perfect, thanks will give it a try!
